release-notes/sql-odbc.OpenSearch.release-notes-1.5.0.0.md

```diff
@@ -0,0 +1,43 @@
+## Version 1.5
+
+### Features
+
+* Add new parameter `tunnelHost` for SIGv4 auth. ([#42](https://github.com/opensearch-project/sql-odbc/pull/42))
+
+### Enhancements
+
+* Fix certificate validation for ODBC driver ([#479](https://github.com/opensearch-project/sql/pull/479))
+* Update SHOW TABLES query. ([#48](https://github.com/opensearch-project/sql-odbc/pull/48))
+
+### Bug Fixes
+
+* ODBC SSL Compliance Fix ([#653](https://github.com/opensearch-project/sql/pull/653))
+* Reverted UseSSL flag to false and removed invalid test case ([#671](https://github.com/opensearch-project/sql/pull/671))
+
+### Documentation
+
+* Add how to setup aws credentials for ODBC Tableau ([#394](https://github.com/opensearch-project/sql/pull/394))
+* Update BI connectors and drivers readme files ([#665](https://github.com/opensearch-project/sql/pull/665))
+
+### Maintenance
+
+* Revert to windows 2019 for odbc CI ([#413](https://github.com/opensearch-project/sql/pull/413))
+* Version bump to 1.3 ([#419](https://github.com/opensearch-project/sql/pull/419))
+* Change ODBC version to 1.4 for release ([#542](https://github.com/opensearch-project/sql/pull/542))
```

Comment:
did we not create release notes for each of these...?

Reply:
No, it wasn't released. It was repo-wide version bump on SQL repo which contained ODBC as well.